pkg/recorder/event.go

```diff
@@ -0,0 +1,25 @@
+package recorder
+
+import (
+	"time"
+
+	"github.com/common-fate/ddb"
+	"github.com/common-fate/granted-approvals/pkg/storage/keys"
+)
+
+type Event struct {
```

Comment:
We should be able to delete this right? I think I added it in my initial prototype before I realised we could reuse the `RequestEvent` struct for it.